packages/runtime/test/src/interpreter/inner-transaction.ts

```diff
@@ -1409,6 +1410,25 @@ describe("Inner Transactions", function () {
 			});
 		});
 
+	});
+
+	describe("Teal v7 update", function () {
```

Comment:
```suggestion
	describe("Teal v7", function () {
```

Reply:
I am not sure why but when I change code like  your suggestion, it appear so much error
<img width="689" alt="image" src="https://user-images.githubusercontent.com/24353881/192527901-22e2ebc1-a1b1-4b10-bff3-7cd5fab6bb91.png">